Formula/dub.rb

```diff
@@ -1,31 +1,37 @@
 class Dub < Formula
   desc "Build tool for D projects"
   homepage "https://code.dlang.org/about"
-  url "https://github.com/dlang/dub/archive/v1.1.2.tar.gz"
-  sha256 "2945c8ab52a421da3ae4c64cc655e3322bd01a9a15ef1ea0208ec999d0e36b91"
-
+  version_scheme 1
```

Comment:
unfortunately I needed to add this to prevent `devel version should not decrease (from 2 to 1.2.1-beta.1)`. Seems that the old 1.2.1-beta.2 was mistakenly detected as version 2.